src/components/Workflow/workflowQueries.ts

```diff
@@ -13,9 +14,10 @@ export function makeWorkflowQuery(queryClient: QueryClient, id: WorkflowId): Que
       // On successful workflow fetch, extract and cache all task templates
       // stored on the workflow so that we don't need to fetch them separately
       // if future queries reference them.
-      extractTaskTemplates(workflow).forEach((task) =>
-        queryClient.setQueryData([QueryType.TaskTemplate, task.id], task),
-      );
+      extractTaskTemplates(workflow).forEach((task) => {
+        return queryClient.setQueryData([QueryType.TaskTemplate, task.id], task);
```

Comment:
What's the purpose of the `return` here?

Reply:
Will remove, was adding `{}` curely braces during debug. Curly braces as not needed anymore and as such return not needed too. 
Will update in a miunte